lib/spack/docs/environments.rst

```diff
@@ -46,8 +46,8 @@ provide some distinctive features:
 
 Spack uses a "manifest and lock" model similar to `Bundler gemfiles
 <https://bundler.io/man/gemfile.5.html>`_ and other package
-managers. The user input file is named ``spack.yaml`` and the lock
-file is named ``spack.lock``
+managers. The user input file is YAML (usually named ``spack.yaml``) and
+the lock file is generated as ``spack.lock``
```

Comment:
I would revert the specific change here, since it seem to describe the structure of an environment. It's the `init` file to `spack create` that can be named differently, but Spack will search for `spack.yaml` and `spack.lock` in a directory to activate an environment.

Reply:
This one :point_up: seems skipped